bin/phpdoc.php

```diff
@@ -1,14 +0,0 @@
-#!/usr/bin/env php
```

Comment:
`./bin/phpdoc.php` has been deprecated in favor of `./bin/phpdoc` since forever; I have now removed it